.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish CDK Manually
+name: Publish Python CDK Manually
```

Comment:
In this original file, I added "Python" everywhere to disambiguate. (Should be only cosmetic changes.)